es6/jsxcad-geometry.js

```diff
@@ -1900,17 +1900,12 @@ const grow = (geometry, amount) => {
   return rewrite(toTransformedGeometry(geometry), op);
 };
 
-// This alphabet uses `A-Za-z0-9_-` symbols. The genetic algorithm helped
-// optimize the gzip compression for this alphabet.
 let urlAlphabet =
   'ModuleSymbhasOwnPr-0123456789ABCDEFGHNRVfgctiUvz_KqYTJkLxpZXIjQW';
-
 let nanoid = (size = 21) => {
   let id = '';
-  // A compact alternative for `for (var i = 0; i < step; i++)`.
   let i = size;
   while (i--) {
```

Comment:
This is interesting. Should I be doing this instead of `for (var i = 0; i < step; i++)`?

Reply:
Well, not really, since it does something else.

As an alternative to for (let i = step; i > 0; i--) { ... } perhaps. :)

But I'd prefer the for statement on the grounds that it limits scope better.